dataset_listing.tsv

```diff
@@ -1,77 +1,80 @@
+ds000248	MNE sample data: Data with visual and auditory stimuli	[link](https://openneuro.org/datasets/ds000248/versions/00001)	[@agramfort](https://github.com/agramfort)	"anat, meg"	"FLASH, T1w, channels, coordsystem, events, meg, scans"
+eeg_ds000117	"Multimodal (fMRI, MEG, EEG) stripped down to EEG with MRI anatomical scan and electrode coordinates. EEGLAB data format (.set, .fdt)"	[link](https://openneuro.org/datasets/ds000117/)	[@robertoostenveld](https://github.com/robertoostenveld)	"anat, eeg"	"T1w, channels, coordsystem, eeg, electrodes, events"
+eeg_rest_fmri	"Resting state with simultaneous fMRI. BrainVision data format (.eeg, .vhdr, .vmrk)"		[@cpernet](https://github.com/cpernet)	"anat, dwi, eeg, func"	"T1w, bold, dwi, eeg"
+ieeg_epilepsy	"multiple sessions, tutorial"	[link](https://neuroimage.usc.edu/bst/getupdate.php?s=tutorial_epimap_bids)	[@ftadel](https://github.com/ftadel)	"anat, ieeg"	"T1w, channels, coordsystem, electrodes, events, ieeg, scans"
+ieeg_epilepsyNWB	"multiple sessions, tutorial — derivative dataset of `ieeg_epilepsy` showcasing the NWB file format alternative"	[link](https://neuroimage.usc.edu/bst/getupdate.php?s=tutorial_epimap_bids)	[@TheChymera](https://github.com/TheChymera)	"anat, ieeg"	"T1w, channels, coordsystem, electrodes, events, ieeg, scans"
+ieeg_epilepsy_ecog	"multiple sessions, tutorial"	[link](https://neuroimage.usc.edu/bst/getupdate.php?s=sample_ecog)	[@ftadel](https://github.com/ftadel)	"anat, ieeg"	"T1w, channels, coordsystem, electrodes, events, ieeg, photo, scans"
+ieeg_visual_multimodal			[@irisgroen](https://github.com/irisgroen)	"anat, fmap, func, ieeg"	"T1w, bold, channels, coordsystem, electrodes, epi, events, ieeg, sbref"
+genetics_ukbb	"multiple tasks, T1w, DTI, BOLD, genetic info"		[@cpernet](https://github.com/cpernet)	"anat, dwi, fmap, func"	"FLAIR, T1w, bold, dwi, events, info, magnitude1, phasediff"
+motion_dualtask	older and younger participants walking while performing discrimination task		[@sjeung](https://github.com/sjeung)	"eeg, motion"	"channels, eeg, events, motion, scans"
+motion_spotrotation	participants rotated heading using full-body motion or joystick	[link](https://openneuro.org/datasets/ds004460)	[@sjeung](https://github.com/sjeung)	"eeg, motion"	"channels, coordsystem, eeg, electrodes, events, motion, scans"
+mrs_2dmrsi	2D sLASER MRSI data from 8 subjects	[link](https://zenodo.org/records/7701228)	[@markmikkelsen](https://github.com/markmikkelsen)	"anat, mrs"	"T1w, mrsi"
+mrs_biggaba	MEGA-PRESS and PRESS MRS data from 12 subjects from one site from the Big GABA project	[link](https://www.nitrc.org/projects/biggaba)	[@markmikkelsen](https://github.com/markmikkelsen)	"anat, mrs"	"T1w, svs, ref"
+mrs_fmrs	Functional MRS data involving a pain stimulus task from 15 subjects	[link](https://www.nitrc.org/projects/fmrs_2020)	[@markmikkelsen](https://github.com/markmikkelsen)	"anat, mrs"	"T1w, svs, ref, events"

```

Comment:
Looks like this file got regenerated by some process. @Remi-Gau do we want to try to clean this up so only changed lines appear, or do we not care?

Reply:
fine with me to leave it as is